kafka/metrics.go

```diff
@@ -246,3 +528,10 @@ func attributesFromRecord(r *kgo.Record, extra ...attribute.KeyValue) []attribut
 	}
 	return attrs
 }
+
+func strnode(node int32) string {
```

Comment:
question: could you clarify why this is needed and why the return value (seems to) differs between `node < 0` and `node >= 0`?